starwars/src/components/StarWars.css

```
@@ -0,0 +1,33 @@
+.buttons {
+    display: flex;
+    justify-content: center;
+}
+
+button {
+    height: 40px;
+    width: 100px;
+    margin-right: 30px;
+}
+
+.left {
+    margin-right: 80px;
+}
+
+.characterContainer {
+    /* border: 1px solid red; */
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.characterCard {
+    border: 1px solid black;
+    background-image: url("http://www.storywarren.com/wp-content/uploads/2016/09/space-1.jpg");
+    width: 400px;
+    margin-top: 20px;
+    margin-right: 40px;
+}
+
+.characterCard h2 {
+    color: yellow;
+}
```

Tiny detail, but this should make your background image nicer. :)
```suggestion
}

body {
  background-attachment: fixed;
}

h1 {
  margin: 0;
  padding: 30px 0;
}
```